src/main/java/seedu/duke/Duke.java

```diff
@@ -41,10 +41,69 @@
         BusStops.PGPR
     };
 
+    private static final BusStops[] BB = {
+        BusStops.PGP,
+        BusStops.KENTRIDGEMRTSTATION,
+        BusStops.LT27,
+        BusStops.UNIVERSITYHALL,
+        BusStops.OPPUNIVERSITYHEALTHCENTRE,
+        BusStops.UNIVERSITYTOWN,
+        BusStops.UNIVERSITYHEALTHCENTRE,
+        BusStops.OPPUNIVERSITYHALL,
+        BusStops.S17,
+        BusStops.OPPKENTRIDGEMRTSTATION,
+        BusStops.PGPR
+    };
+
+    private static final BusStops[] CC = {
+        BusStops.PGPR,
+        BusStops.TCOMS,
+        BusStops.OPPHSSML,
+        BusStops.OPPNUSS,
+        BusStops.COM2,
+        BusStops.VENTUS,
+        BusStops.NUSIT,
+        BusStops.OPPYUSOFISHAKHOUSE,
+        BusStops.UNIVERSITYTOWN,
+        BusStops.YUSOFISHAKHOUSE,
+        BusStops.CENTRALLIBRARY,
+        BusStops.LT13,
+        BusStops.AS5,
+        BusStops.COM2,
+        BusStops.BIZ2,
+        BusStops.OPPTCOMS,
+        BusStops.PGP,
+        BusStops.OPPKENTRIDGEMRTSTATION
+    };
+
+    private static final BusStops[] CCX = {
+        BusStops.OPPKENTRIDGEMRTSTATION,
+        BusStops.UNIVERSITYTOWN,
+        BusStops.OPPKENTRIDGEMRTSTATION
+    };
+
+    private static final BusStops[] DD1 = {
+        BusStops.OEITIONGHAM,
+        BusStops.BOTANICGARDENSMRT,
+        BusStops.UNIVERSITYTOWN,
+        BusStops.KENTVALE
+    };
+    private static final BusStops[] DD2 = {
+        BusStops.KENTVALE,
+        BusStops.UNIVERSITYTOWN,
+        BusStops.COLLEGEGREEN,
+        BusStops.OEITIONGHAM
+    };
+
```

Comment:
Wondering if we want to move it to a different class like #57

Reply:
sure why not